spec/inputs/snmp_spec.rb

```diff
@@ -54,13 +54,15 @@
           {"get" => ["1.0"], "hosts" => [{"host" => "udp:localhost/161"}]},
           {"get" => ["1.0"], "hosts" => [{"host" => "udp:127.0.0.1/112345"}]},
           {"get" => ["1.0"], "hosts" => [{"host" => "udp:127.0.0.1/161", "community" => "public"}]},
+          {"get" => ["1.0"], "hosts" => [{"host" => "tcp:127.0.0.1/112345"}]},
+          {"get" => ["1.0"], "hosts" => [{"host" => "tcp:127.0.0.1/161", "community" => "public"}]},
```

Comment:
should't this be 1161?

Reply:
doesn't have to be - note that we are only verifying the format here but in any case, the port number can be any number.

Reply:
right, I was just aiming at consistency in the examples that reference 1161 for tcp in two other places.